spec/factories/users.rb

```diff
@@ -18,7 +18,7 @@
 #  last_request_at        :datetime
 #  last_sign_in_at        :datetime
 #  last_sign_in_ip        :inet
-#  name                   :string           default("CHANGEME"), not null
+#  name                   :string           default("Unnamed"), not null
```

Comment:
Let's update the schema & the factory annotations to match this change.

Reply:
Updated!